android/app/src/main/java/org/fossasia/openevent/activities/auth/EditProfileActivity.java

```diff
@@ -230,20 +234,27 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
 
             imageUri = data.getData();
             Timber.d(imageUri.toString());
+            UCrop.of(imageUri,Uri.fromFile(new File(getCacheDir(),imageUri.getUserInfo()+".png"))).start(this);
 
+        }
+        if (resultCode == RESULT_OK && requestCode == UCrop.REQUEST_CROP) {
+            final Uri resultUri = UCrop.getOutput(data);
             InputStream imageStream = null;
             try {
-                imageStream = getContentResolver().openInputStream(imageUri);
+                imageStream = getContentResolver().openInputStream(resultUri);
             } catch (FileNotFoundException e) {
                 e.printStackTrace();
             }
             Bitmap selectedImage = BitmapFactory.decodeStream(imageStream);
             encodedImage = encodeImage(selectedImage);
 
             OpenEventApp.picassoWithCache
-                    .load(imageUri)
+                    .load(resultUri)
                     .transform(new CircleTransform())
                     .into(avatar);
+        } else if (resultCode == UCrop.RESULT_ERROR) {
+            final Throwable cropError = UCrop.getError(data);
+            Timber.d("EditProfileActivity", "UCrop Error"+cropError );
```

Comment:
Spaces around `+` and no space before `)`

Reply:
@iamareebjamal Check gitter plz 👍